src/devices/SensorHub/README.md

```diff
@@ -0,0 +1,10 @@
+﻿# SensorHub
```

Comment:
please edit this line to: `SensorHub - <some short description>` - we use this line to generate https://github.com/dotnet/iot/blob/main/src/devices/README.md

Reply:
done